src/js/components/pipelines/DataTable.jsx

```diff
@@ -175,7 +180,7 @@ var DataTable = React.createClass({
                     onMouseOver={this.showFullField}>{k}</td>
                   {values.map(function(v, i) {
                     return (
-                      <td key={v._id} className={i % 2 ? 'even' : 'odd'}
+                      <td key={k + i} className={i % 2 ? 'even' : 'odd'}
```

Comment:
@arvind After dropping a field onto a channel, `v._id` is undefined on the next run. This fix suppresses the react error but I did not look into why `._id` is being unset.